VideoAPI/Video.API/Controllers/AudioRecordingController.cs

```diff
@@ -173,7 +173,7 @@ public async Task<IActionResult> GetAudioStreamMonitoringInfoAsync(Guid hearingI
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
         public async Task<IActionResult> GetAudioRecordingLinkAsync(Guid hearingId)
         {
-            _logger.LogInformation($"Getting audio recording link for hearing: {hearingId}");
+            _logger.LogInformation("Getting audio recording link");
             try
```

Comment:
Did you mean to remove hearing id from the log altogether (i.e. not even as an argument)?

Reply:
theyre added to the scope elsewhere so will still appear in properties, i thought by removing we could easier group logs easier

Reply:
Never mind, I saw the implementation of the data extractor